From 096d50d1c983222d986b625ca534784b3525b616 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 19 Sep 2020 20:03:09 +0200
Subject: [PATCH 1/1] MAINTAINERS: assign doc/arch/sandbox.rst

Add doc/arch/sandbox.rst to the scope of SANDBOX.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 7e46470c70..a8d8017b5f 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -951,6 +951,7 @@ SANDBOX
 M:	Simon Glass <sjg@chromium.org>
 S:	Maintained
 F:	arch/sandbox/
+F:	doc/arch/sandbox.rst
 
 SH
 M:	Marek Vasut <marek.vasut+renesas@gmail.com>
-- 
2.28.0

